misk-hibernate/src/main/kotlin/misk/hibernate/ReflectionQueryFactory.kt

```diff
@@ -68,7 +72,9 @@ internal class ReflectionQuery<T : DbEntity<T>>(
 
     val typedQuery = session.hibernateSession.createQuery(query)
     typedQuery.maxResults = effectiveMaxRows(returnList)
-    val rows = typedQuery.list()
+    val rows = traceSelect {
+      typedQuery.list()
```

Comment:
is there a preload hook we could include in QueryTracingListener instead of having a separate place where queries are traced here?

Reply:
there is, it doesn't do what we need. I have added a comment about it in `QueryTracingListener`.